pay-api/src/pay_api/templates/credit_add_notification.html

```diff
@@ -0,0 +1,15 @@
+# ${{ amount }} credit added to your account
+
+We've added ${{ amount }} credit to your account:
```

Comment:
isn't this the same thing repeated?

Reply:
@Jxio

Reply:
![image](https://github.com/user-attachments/assets/435c6218-9755-4ce9-bb61-d90520df93f2)